apps/wallet/src/ui/app/hooks/useGetAllBalances.ts

```diff
@@ -8,6 +8,7 @@ import { useQuery } from '@tanstack/react-query';
 
 import { FEATURES } from '_src/shared/experimentation/features';
 
+// TODO: use from core. Update core to handle optional select function
```

Comment:
Would this be a follow up PR or missed it?

Reply:
Updated this PR to resolve this